src/components/card/demo/cloudy.twig

```diff
@@ -0,0 +1,26 @@
+{% embed '@cloudfour/components/card/card.twig' with {
+  tag_name: 'aside',
+  class: 'c-card--contained c-card--cloudy t-dark'
+} only %}
+  {% block content %}
+    <div class="o-rhythm o-rhythm--compact has-big-font-size">
```

Comment:
Look, I know `has-big-font-size` is a WordPress utility class, I know I probably _should_ add an equivalent for our project. But this works _now_, and perfect is the enemy of good... right? Right???